src/main/webapp/app/shared/markdown-editor/monaco/markdown-editor-monaco.component.html

```diff
@@ -1,134 +1,130 @@
-<div
-    #wrapper
-    class="markdown-editor-wrapper"
-    [style.overflow]="inPreviewMode ? 'auto' : 'hidden'"
-    [style.height.px]="inPreviewMode ? undefined : targetWrapperHeight"
-    [style.height.%]="targetWrapperHeight ? undefined : 100"
-    [style.cursor]="isResizing ? 'ns-resize' : undefined"
->
-    <nav ngbNav #actionPalette #nav="ngbNav" class="nav-tabs" [destroyOnHide]="false" (navChange)="onNavChanged($event)">
-        <!-- TODO if show edit -->
-        <ng-container ngbNavItem="editor_edit">
-            <a ngbNavLink class="btn-sm text-normal px-2 py-0 m-0"><span jhiTranslate="entity.action.edit"></span></a>
-            <ng-template ngbNavContent>
-                <div class="markdown-editor markdown-editor-wrapper d-flex flex-column">
-                    <jhi-monaco-editor class="h-100" [shrinkToFit]="false" [stickyScroll]="false" (textChanged)="onTextChanged($event)" (drop)="onFileDrop($event)" />
-                    @if (enableFileUpload) {
-                        <div class="d-flex" #fileUploadFooter>
-                            <input
-                                [id]="'file-upload ' + uniqueMarkdownEditorId"
-                                class="file-upload-footer-input"
-                                type="file"
-                                accept="image/*"
-                                [multiple]="true"
-                                (change)="onFileUpload($event)"
-                            />
-                            <label
-                                [for]="'file-upload ' + uniqueMarkdownEditorId"
-                                class="file-upload-footer-label d-inline"
-                                [style.cursor]="isResizing ? 'ns-resize !important' : undefined"
-                            >
-                                <div class="row mx-0 align-items-baseline">
-                                    <span class="col upload-subtitle" jhiTranslate="artemisApp.markdownEditor.fileUpload"></span>
-                                    <a class="col-auto" href="http://demo.showdownjs.com">
-                                        <fa-icon [icon]="faQuestionCircle" [ngbTooltip]="'artemisApp.markdownEditor.guide' | artemisTranslate" />
-                                    </a>
-                                </div>
-                            </label>
-                        </div>
-                    }
-                    @if (enableResize) {
-                        <!-- This placeholder will hold the resize handle. -->
-                        <span #resizablePlaceholder></span>
-                    }
-                </div>
```

Comment:
The diff is a little hard to read here. Basically, I removed this part and wrapped the #wrapper div in a #fullElement div. I then added the #resizePlaceholder as a separate div below the #wrapper but inside the #fullElement